### DISCUSSION TITLE
Understanding breaking changes for 0.30.0

### DISCUSSION THREAD

**Discussion by dchicchon** [ramda/ramda#3471] [Q&A] [answered]
Currently I'm upgrading my system from 0.29.0 to 0.30.0. During the upgrade process I viewed the changes provided in the [upgrade guide](https://github.com/ramda/ramda/discussions) and I see there is a breaking change listed:

💥 Breaking Changes:
https://github.com/ramda/ramda/pull/3388: use === to test for placeholder

Can someone explain exactly what this breaking change implies to using ramda itself? From what I can tell, it looks like the equality operator has changed in the `isPlaceholder` method but that's all I see.

**Comment by kedashoe** [comment, marked as answer]
This was an internal detail, but has [been reverted](https://github.com/ramda/ramda/pull/3463) as it can cause issues when different versions of ramda are used in the same project.

**Reply by dchicchon** [reply]
Understood, so just to be clear this breaking change is not in the latest build npm package correct? Or will this revert be applied to the next patch bump?

**Reply by kedashoe** [reply]
It will be reverted in the next patch release. I'll try to get 0.30.1 out this weekend.
